sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcTest.scala

```diff
@@ -106,4 +106,14 @@ abstract class OrcTest extends QueryTest with SQLTestUtils with BeforeAndAfterAl
       df: DataFrame, path: File): Unit = {
     df.write.mode(SaveMode.Overwrite).orc(path.getCanonicalPath)
   }
+
+  protected def checkPredicatePushDown(df: DataFrame, numRows: Int, predicate: String): Unit = {
```

Comment:
@HyukjinKwon . I refactor this since it's repeated three times now.
And, this function should be here because the existing two instances are in `OrcQueryTest` and new instance is in `OrcQuerySuite`.